database/seeders/GamesTableSeeder.php

```diff
@@ -43,7 +44,10 @@ public function run(): void
          * add achievements to games
          */
         Game::all()->each(function (Game $game) {
-            $game->achievements()->saveMany(Achievement::factory()->count(random_int(0, 10))->create());
+            $game->achievements()->saveMany(Achievement::factory()->count(random_int(0, 10))->create([
+                'GameID' => $game->ID,
+                'Flags' => AchievementType::OfficialCore,
+            ]));
```

Comment:
These achievements should be assigned to valid users (preferably only one user for most games, and 2-4 on a handful of others). They're currently generating unique user names, so all the achievements are owned by unique non-existent users:

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9bfe232f-e09a-4205-a3cb-7fcb01f3b7b7)

Reply:
Sounds good, thanks for looking into it this closely! 
The users seeds include one user per upcoming permission matrix role - we could reuse those for the various author relations, too. I'll have a look at that another time - the seeders need more love in general.